packages/help-center/src/hooks/use-zendesk-config.ts

```diff
@@ -0,0 +1,18 @@
+import { useQuery } from '@tanstack/react-query';
+
+async function requestZendeskConfig() {
+	return await window.fetch( 'https://wpcom.zendesk.com/embeddable/config', {
+		method: 'GET',
```

Comment:
Default method is `GET`.
```suggestion
		
```

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/76777/commits/bcbcc946e4440b6c606700fea168825b586e6007